lib/src/components/NxTag/NxTag.scss

```diff
@@ -33,25 +33,33 @@
 .nx-tag--selectable {
   cursor: pointer;
   padding-right: var(--nx-spacing-2x);
-
   background-color: var(--nx-selectable-color-light);
 
   &:hover {
     border-color: var(--nx-color-interactive-border-hover);
   }
 
   &:focus {
-    border: 1px solid var(--nx-color-interactive-border-focus);
-    box-shadow: var(--nx-box-shadow-focus);
-    outline: none;
+    &:focus-visible {
```

Comment:
I'm not under the impression that this extra bit of selector is needed here.  Disabling the native outline works directly on `:focus`